### PR TITLE
Support ExactInOrOut

### PR DESCRIPTION
**Pull request by worlddlckgh**

### CODE REVIEW

File: src/components/Form.tsx [hunk 1]
```diff
@@ -281,13 +300,25 @@ const Form: React.FC<{
                         thousandSeparator={thousandSeparator}
                         allowNegative={false}
                         valueIsNumericString
-                        onValueChange={({ value }) => onChangeToValue(value)}
-                        placeholder={swapMode === 'ExactOut' ? 'Enter desired amount' : ''}
+                        onValueChange={onChangeToValue}
+                        placeholder={swapMode === 'ExactIn' ? '' : 'Enter desired amount'}
```
Comment (wkitsin) [inline src/components/Form.tsx:304]: why is this changed a ? if im in exactIn mode, to token info should not have placeholder right ? 
CleanShot 2024-06-27 at 6  59 09

File: src/constants/index.ts [hunk 2]
```diff
@@ -38,7 +37,7 @@ export const INITIAL_FORM_CONFIG: IFormConfigurator = Object.freeze({
   formProps: {
     fixedInputMint: false,
     fixedOutputMint: false,
-    swapMode: SwapMode.ExactIn,
+    swapMode: 'ExactInOrOut' as IFormConfigurator['formProps']['swapMode'],
```
[the remark this answers is not in the document]
Comment (worlddlckgh) [reply src/constants/index.ts:40]: not sure how this works, but casting swapMode as const works
